integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -365,6 +435,151 @@ export class CdcJsService implements OnDestroy {
     }
   }
 
+  /**
+   * Trigger CDC user email update.
+   *
+   * @param password
+   * @param newEmail
+   */
+  updateUserEmailWithoutScreenSet(
+    password: string,
+    newEmail: string
+  ): Observable<{ status: string }> {
+    if (
+      !password ||
+      password?.length === 0 ||
+      !newEmail ||
+      newEmail?.length === 0
+    ) {
+      return throwError('Email or password not provided');
+    } else {
+      //Verify the password by attempting to login
+      return this.getLoggedInUserEmail().pipe(
+        switchMap((user) => {
+          let email = user?.uid;
+          if (!email || email?.length === 0) {
+            return throwError('Email or password not provided');
+          }
+          // Verify the password by attempting to login
+          // - CDC doesn't require to verify password before changing an email, but the default Spartacus requires it.
+          // - CDC doesn't have any specific api, for verifying a password, so as a _workaround_ we call the login API of CDC.
+          //   We pass a special `context` parameter `'{ skipOccAuth: true}'`, which doens't mean instruct the CDC API to reset email,
+          //   but only conditionally avoid the full CDC login flow.
```

Comment:
Since we renamed the context  from `RESET_EMAIL` to `{skipOccAuth:true}`, part of this code comment is not needed anymore:

```suggestion
          //   We pass a special `context` parameter `'{ skipOccAuth: true }'`
          //   to avoid the full CDC login flow.
```